middlewares/validators/recruiter.js

```diff
@@ -0,0 +1,27 @@
+const joi = require('joi')
+
+const validateRecruiter = async (req, res, next) => {
+  const schema = joi.object().keys({
+    company: joi.string().optional(),
+    first_name: joi.string().optional(),
+    last_name: joi.string().optional(),
+    designation: joi.string().optional(),
+    reason: joi.string().optional(),
+    email: joi.string().optional(),
+    currency: joi.string().optional(),
+    package: joi.number().optional(),
+    timestamp: joi.string().optional()
```

Comment:
I have a doubt regarding these fields. Why are we having all of them optional ?

Reply:
- Every field except the package should be `required`
- Because other fields are important and as per the [Issue](https://github.com/Real-Dev-Squad/website-members/issues/127 "Recruiter Issue")